### DISCUSSION TITLE
pag运行在部分安卓手机上动画会花屏

### DISCUSSION THREAD

**Discussion by demonnboy** [Tencent/libpag#2295] [Q&A]
【版本信息】
libpag:4.3.48' | Android 10 | PAGViewer 版本 | AE 版本: 2021。

【平台信息】
RN运行在安卓手机
Android版本:10
手机:IQOO

【预期的表现】
正常执行pag动画
【实际的情况】
中间的小猪是pag动画在执行loop=0时会出现花屏的情况
https://github.com/Tencent/libpag/assets/12589612/bebb1144-255e-4fb4-bf01-a892778fd1fb

【Demo及附件】
<SHRCTPagPlayer style={{ width: shihuo.aiDp(247), height: shihuo.aiDp(239) }}
                        source={'http://static.shihuocdn.cn/app/res/dsgn/c861381b5c7348df9ce825cd55526178.pag'}
                        loops={0}
                        speed={1}
                        ref={ref => this.smallPigRef = ref}
                    />

RN封装的原生组件

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

因长期无回复，且 libpag 4.3 及更早版本已停止维护（当前仅维护 4.4 / 4.5），先关闭本条。
上报版本：4.3.48。

如果问题在最新版本（4.4 / 4.5）仍能复现，欢迎在本帖回复或新开 issue，并附上 libpag 版本号、平台与系统版本、可复现的 PAG 文件或最小 demo，我们会优先跟进。（stale-close）
